css/css-text/text-indent/reference/text-indent-each-line-hanging-ref.html

```diff
@@ -0,0 +1,47 @@
+<html>
```

Comment:
To be consistent with the test file:
 
```suggestion
<!DOCTYPE html>
<html>
```